app/jobs/characterize_job.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+# OVERIDE Hyrax 3.6 to support fits servlet
+class CharacterizeJob < Hyrax::ApplicationJob
+  queue_as Hyrax.config.ingest_queue_name
+
+  # Characterizes the file at 'filepath' if available, otherwise, pulls a copy from the repository
+  # and runs characterization on that file.
+  # @param [FileSet] file_set
+  # @param [String] file_id identifier for a Hydra::PCDM::File
+  # @param [String, NilClass] filepath the cached file within the Hyrax.config.working_path
+  def perform(file_set, file_id, filepath = nil)
+    # rubocop:disable Metrics/LineLength
+    raise "#{file_set.class.characterization_proxy} was not found for FileSet #{file_set.id}" unless file_set.characterization_proxy?
+    filepath = Hyrax::WorkingDirectory.find_or_retrieve(file_id, file_set.id) unless filepath && File.exist?(filepath)
+    Hydra::Works::CharacterizationService.run(file_set.characterization_proxy, filepath, ch12n_tool: :fits_servlet)
```

Comment:
Reading note: consider the method signature later; namely how we've modified it.